### PR TITLE
Add TextAlign and Padding options

### PR DESCRIPTION
**Pull request by wer-mathurin**

### CODE REVIEW

**Comment by wer-mathurin** [comment]
@Frezyx Can you look at those proposed changes, they are not breaking changes, just more options :-)

File: lib/group_button.dart [hunk 1]
```diff
@@ -33,13 +33,21 @@ class GroupButton extends StatelessWidget {
...
           (isRadio && selectedButtons == null) ||
               (!isRadio && selectedButton == null),
           "You can use selectedButton field for isRadio [true] and selectedButtons field with isadio [false]",
         ),
         super(key: key);
 
+  /// [EdgeInsets] the text alignment to use in the buttons for the [GroupButton]
+  final EdgeInsets? textPadding;
+
+  /// [TextAlign] the text alignment to use in the buttons for the [GroupButton]
```
Comment (Frezyx) [inline lib/group_button.dart:48]: buttons text alignment

File: lib/group_button.dart [hunk 2]
```diff
@@ -33,13 +33,21 @@ class GroupButton extends StatelessWidget {
...
+    this.textAlign,
+    this.textPadding,
   })  : assert(
           (isRadio && selectedButtons == null) ||
               (!isRadio && selectedButton == null),
           "You can use selectedButton field for isRadio [true] and selectedButtons field with isadio [false]",
         ),
         super(key: key);
 
+  /// [EdgeInsets] the text alignment to use in the buttons for the [GroupButton]
```
Comment (Frezyx) [inline lib/group_button.dart:45]: inner padding of buttons

**Review by Frezyx — CHANGES_REQUESTED** [review]
Hello @wer-mathurin !
Thank you for propose this changes. 

I will merge it when you fix my review comments

**Comment by Frezyx** [comment]
And if it's not difficult for you
Add this new fields in [README](https://github.com/Frezyx/group_button/blob/master/README.md) file

**Comment by wer-mathurin** [comment]
@Frezyx all request resolve in the last commit

**Review by Frezyx — APPROVED** [review]

**Comment by Frezyx** [comment]
Thank you for your contribution !

**Pull request merged**